connector/src/main/java/org/geysermc/connector/network/translators/world/block/entity/EndGatewayBlockEntityTranslator.java

```diff
@@ -39,7 +39,9 @@
 public class EndGatewayBlockEntityTranslator extends BlockEntityTranslator {
     @Override
     public void translateTag(NbtMapBuilder builder, CompoundTag tag, int blockState) {
-        builder.put("Age", (int) ((long) tag.get("Age").getValue()));
+        if(tag.contains("Age")) {
```

Comment:
Can you change this to something like ` Tag ageTag = tag.get("Age")` and then do an `ageTag instanceof LongTag`?